packages/core/src/components/tree/tree.tsx

```diff
@@ -71,6 +75,14 @@ export class Tree<T = {}> extends React.Component<ITreeProps<T>, {}> {
         );
     }
 
+    public nodeFromPath(path: number[]): ITreeNode<T> {
```

Comment:
but actually remove this. we already have the static method.

Reply:
oh sounds good, didn't see the hidden comment :P